metricflow/test/fixtures/source_table_snapshots/simple_model/dim_listings.yaml

```diff
@@ -0,0 +1,29 @@
+---
+table_snapshot:
+  table_name: dim_listings
+  column_definitions:
+    - name: active_from
+      type: TIME
+    - name: active_to
+      type: TIME
+    - name: listing_id
+      type: STRING
+    - name: country
+      type: STRING
+    - name: capacity
+      type: INT
+    - name: is_lux
+      type: BOOLEAN
+    - name: user_id
+      type: STRING
+  rows:
+    - ["2020-01-01", "2020-01-02", "l3141592", "us", "2", "True", "u0004114"]
```

Comment:
How did you move all of these? Manually?

Reply:
I wrote a script that takes care of the bulk of the work in migrating them.